tomopy/prep/phase.py

```diff
@@ -137,26 +138,20 @@ def retrieve_phase(
 def _retrieve_phase(tomo, phase_filter, px, py, prj, pad):
     dx, dy, dz = tomo.shape
     num_jobs = tomo.shape[0]
+    fftw_effort = 'FFTW_MEASURE' if num_jobs > 10 else 'FFTW_ESTIMATE'
     for m in range(num_jobs):
         prj[px:dy + px, py:dz + py] = tomo[m]
         fproj = pyfftw.interfaces.numpy_fft.fft2(
-            prj, planner_effort=_plan_effort(num_jobs))
+                prj, planner_effort=fftw_effort)
         filtproj = np.multiply(phase_filter, fproj)
         proj = np.real(pyfftw.interfaces.numpy_fft.ifft2(
-            filtproj, planner_effort=_plan_effort(num_jobs))
+            filtproj, planner_effort=fftw_effort)
             ) / phase_filter.max()
         if pad:
             proj = proj[px:dy + px, py:dz + py]
         tomo[m] = proj
 
 
-def _plan_effort(num_jobs):
```

Comment:
I switched the function to a statement: `fftw_effort = 'FFTW_MEASURE' if num_jobs > 10 else 'FFTW_ESTIMATE'` (line 141 in phase.py and 132 in stripe.py).
This was to simplify the code (reduce function call and imports).
But you have a point that in terms of maintenance it is better to have it in one place.